homeassistant/components/insteon/config_flow.py

```diff
@@ -306,7 +313,24 @@ async def async_step_change_hub_config(self, user_input=None):
             step_id=STEP_CHANGE_HUB_CONFIG, data_schema=data_schema
         )
 
-    async def async_step_add_override(self, user_input=None):
+    async def async_step_change_plm_config(self, user_input=None) -> FlowResult:
+        """Change the PLM configuration."""
+        if user_input is not None:
+            data = {
+                **self.config_entry.data,
+                CONF_DEVICE: user_input[CONF_DEVICE],
+            }
+            self.hass.config_entries.async_update_entry(self.config_entry, data=data)
+            return self.async_create_entry(
+                title="",
+                data={**self.config_entry.options},
+            )
+        data_schema = build_plm_schema(**self.config_entry.data)
```

Comment:
Would it be possible to list all serial devices here? Maybe even filtered by pid?

Reply:
ok to do in a future PR.